File: python/test_client.py
import tcp_protocol_client as tcc

HOST = 'localhost'
PORT = 9999

with tcc.tcp_client(HOST, PORT) as tccp:
    for i in range(1000):
        k, l = tccp.recv()
        print(l)
        # k, l = tccp.recv()
        # print(l)
        tccp.send("ddddddddddddddddddddddddddddddddddddddddddddddd")
    

File: cpp/test_client.cpp
#include "tcp_protocol_client.h"
#include <string>
#include <iostream>
using namespace std;

const string host = "127.0.0.1";
const int port = 9999;

int main()
{
    tcp_client tccp = tcp_client(host, port);
    for(int i = 0; i < 10; i++)
    {
        int k = tccp.Recv<int>();
        cout << k << endl;

        tccp.Send<int>(i);
    }
    
}